server/manager/src/main/java/org/apache/accumulo/manager/Manager.java

```diff
@@ -328,8 +328,8 @@ synchronized void setManagerState(final ManagerState newState) {
         break;
       case HAVE_LOCK:
         if (isUpgrading()) {
+          upgradeCoordinator.continueUpgrade();
           upgradeCoordinator.preUpgradeValidation();
```

Comment:
Wonder if this could also be called in the new pre upgrade step.

Reply:
This reUpgradeValidation( check runs in the manager and can cause upgrade to fail to start.  This failure could also be detected on the command line, letting a user know that the upgrade will fail.  Not a change for this PR, was wondering if it made even made sense to call this code in both places.

Reply:
I see

Reply:
Added this in 376bd50